raj_chetan/algorithms/10-6-16.js

```diff
@@ -0,0 +1,52 @@
+# Parentheses Valid
```

Comment:
good. use the templates file I provided here: https://github.com/dallaspythondojo/python/tree/master/Apatira_Authman/Algorithms

Also, be sure to run your functions to test them, by outputing whatever they return to console.log()